docutils.conf

```diff
@@ -15,7 +15,7 @@ template: pyramid-pep-template
 embed-stylesheet: 0
 
 # path to PEPs, for template:
-pep-home: /dev/peps/
+pep-home: /en/latest/
```

Comment:
That's going to break https://www.python.org/dev/peps/ rending, no?

Reply:
This was a workaround hack to get read the docs working. If #3 (peps.py.org) is tied into this work, then I would suggest a mass 301 moved permanently redirect or similar to whatever the new endpoints are. If this is to be integrated into py.org then as the build process (package.py) transforms the HTML I'm less sure this would be an issue, but broadly the 'right' thing to do with it depends where this piece of work goes (if anywhere!)

Reply:
It turns out removing it entirely works better as docutils then uses the immediate parent dir as the root, so I have done this.